lib/src/main/java/co/nimblehq/alpine/lib/mrz/MrzProcessor.kt

```diff
@@ -1,8 +1,15 @@
 package co.nimblehq.alpine.lib.mrz
 
+import co.nimblehq.alpine.lib.model.*
+import com.google.mlkit.vision.common.InputImage
+
 interface MrzProcessor {
     fun processImageFile(filePath: String, mrzProcessorResultListener: MrzProcessorResultListener)
 
+    fun processImage(cameraImage: CameraImage, mrzProcessorResultListener: MrzProcessorResultListener)
+
+    fun processImage(inputImage: InputImage, mrzProcessorResultListener: MrzProcessorResultListener)
```

Comment:
@ryan-conway Do we still need this method? I didn't see where it use it 🤔

Reply:
@Wadeewee Since it's a library it seems like a good idea to provide options to the end users for how they want to process their images. They may already have an `InputImage` from the `MLKit` library or they may have mapped it to a `CameraImage` from our library 🤔

Reply:
That's a good point 👍 Thanks for your explanation 🙏